mlrun/config.py

```diff
@@ -232,6 +232,7 @@
                 "abort_grace_period": "10",
                 "delete_project": "900",
                 "delete_function": "900",
+                "model_endpoint_creation": "360",
```

Comment:
I suggest increasing it to 10min. Based on your findings, it took about 6 min to create 4k meps, so it makes sense that it might take a bit longer.
```suggestion
                "model_endpoint_creation": "600",
```